src/plugins/gc-checklist/_base.scss

```diff
@@ -0,0 +1,115 @@
+/*
+ *
+ * Web Experience Toolkit (WET) / Boîte à outils de l'expérience Web (BOEW)
+ * wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ *
+ *
+ *	Provisional GC Checklist
+*/
+
+.provisional {
+
+	&.gc-checklist {
+		font-size: 20px;
+		margin-left: 0;
+		padding-top: 6px;
+		div {
+			cursor: default;
+			display: inline-block;
+			padding-left: 15px;
+			position: relative;
+
+			&::before,
+			&::after {
+				left: 0;
+				position: absolute;
+				top: 0;
+				transform: translate(-100%, 0%);
+			}
+
+			&::before {
+				border: 4px solid #fff;
+				box-shadow: 0 0 0 2px #000;
+				content: "";
+				display: inline-block;
+				height: 36px;
+				width: 36px;
+			}
+
+			&::after {
+				content: none;
+			}
+
+			&:focus {
+				outline: none;
+			}
+
+			&:checked::after {
+				content: "";
+			}
+
+			&:active::before,
+			&[aria-checked="false"]:hover::before {
+				background-image: linear-gradient(to bottom, hsl(300, 3%, 73%), hsl(300, 3%, 93%) 30%);
+			}
+
+			&[aria-checked="true"] {
+
+				&::before {
+					background-color: #318000;
+					border: none;
+				}
+
+				&::after {
+					border-color: white;
+					border-style: solid;
+					border-width: 0 5px 5px 0;
+					content: "";
+					display: inline-block;
+					height: 26px;
+					transform: translate(-185%, 10%) rotate(45deg);
+					width: 13px;
+				}
+			}
+
+			&:focus::before {
+				box-shadow: 0 0 0 2px #000, 0 0 8px 4px rgb(59, 153, 252);
+			}
+		}
+
+		.lst-none ul li {
+			list-style-type: disc;
+
+		}
+		.feedbackElm {
+			margin-left: -36px;
+			.label {
+				font-size: 90%;
+			}
+		}
+	}
+
+	&.gc-checklist-sm {
+		font-size: 17px;
+
+		[role="checkbox"] {
+
+			&::before {
+				height: 24px;
+				width: 24px;
+			}
+
+			&[aria-checked="true"]::after {
+				border-width: 0 4px 4px 0;
+				height: 16px;
+				width: 9px;
+			}
+		}
```

Comment:
Don't style aria code, there is even not an example.

Reply:
You're right this file should be removed now.